frameworks/helloworld/src/test/java/com/mesosphere/sdk/helloworld/scheduler/ServiceTest.java

```diff
@@ -389,10 +389,6 @@ public void testWorldDecommission() throws Exception {
         ticks.add(Send.taskStatus("world-0-server", Protos.TaskState.TASK_RUNNING).setReadinessCheckExitCode(0).build());
         ticks.add(Send.taskStatus("world-1-server", Protos.TaskState.TASK_RUNNING).setReadinessCheckExitCode(0).build());
 
-        // An initial kill is sent early on for the decommissioned tasks:
```

Comment:
Going by how the test was already "verifying" the bad behavior during decommission, it looks like this has been going on for a while. I think it's a good sign that this is the only noticeable side effect of cleaning up the old code.